docker-compose.diffy.yaml

```diff
@@ -21,6 +21,7 @@ services:
       - DDEV_HOSTNAME
       - DDEV_UID
       - DDEV_GID
```

Comment:
These two variables are also no longer needed inside the container. (These were used for the `post-start` hook only.)

```suggestion
```